docs/open-source-project/release-notes/tizen-4-0-m2.md

```diff
@@ -26,10 +26,10 @@ The Tizen 4.0 Public M2 release provides developers with the Tizen kernel, devic
 - System Framework
   - Extended internal storage support, without on-demand device encryption, has been added:
     - Extended internal storage is an SD card-based storage extension feature.
-  - Dbus debugging and profiling tools (eBPF-based monitoring) have been added.
-  - Supplementary group support for libdbuspolicy with Kdbus backend has been added.
-  - C# Public APIs (device, feedback, storage, dlog, tizen-platform-config (internal)) has been added.
-  - New Public enumerators have been added to libsvi, such as FEEDBACK_PATTERN_SYSTEM_SHORT.
+  - D-Bus debugging and profiling tools (eBPF-based monitoring) have been added.
+  - Supplementary group support for libdbuspolicy with KDBus backend has been added.
+  - C# API set (device, feedback, storage, dlog, tizen-platform-config (internal)) has been added.
```

Comment:
Why "Public" in this line and line 32 are removed?

Reply:
added